src/pybind/mgr/dashboard/frontend/src/app/shared/api/host.service.spec.ts

```diff
@@ -44,14 +44,15 @@ describe('HostService', () => {
   });
 
   it('should update host', fakeAsync(() => {
-    service.update('mon0', true, ['foo', 'bar']).subscribe();
+    service.update('mon0', true, ['foo', 'bar'], true, false, true).subscribe();
     const req = httpTesting.expectOne('api/host/mon0');
     expect(req.request.method).toBe('PUT');
     expect(req.request.body).toEqual({
       force: false,
       labels: ['foo', 'bar'],
-      maintenance: false,
-      update_labels: true
+      maintenance: true,
```

Comment:
Shouldn't this test be split into 2: 1 for testing regular update and 1 for testing drain?
It seems the maintenance changes...

Reply:
Done!